msp430-gcc.rb

```diff
@@ -30,7 +30,8 @@ def install
     # which, upon further inspection, arises when xgcc bails out when it sees
     # this argument.
     ENV.remove_from_cflags '-Qunused-arguments'
-    ENV.remove_from_cflags '-march=native'
+    ENV.remove_from_cflags(/ ?-march=\S*/)
+    ENV.remove_from_cflags(/ ?-msse4(\.\d)?/)
```

Comment:
Thanks; seeing where these comes from helps. I'll make a small tweak to catch any SSE flag and call it done. Thanks again.